### ISSUE TITLE
ANMN Wave collection filter and parameter issues

### ISSUE THREAD

**Issue by mhidas** [aodn/content#303]
Two issues with the [Australian National Mooring Network (ANMN) - Wave time-series](https://portal.aodn.org.au/search?uuid=aaebf991-b79d-4670-a1c5-a0de9bf649ce) collection:

1) There is a filter for "Sea Surface Wave Significant Height", but this is based on a specific netCDF _variable name_, not the variable's _standard_name_. About half the data records significant wave height under a different variable name, so selecting this filter incorrectly de-selects these data. This collection is all about wave parameters, so I don't think it needs parameter filters at all.

2) Collection also contains temperature harvested from a "TEMP" variable in files from ADCP instruments, which _not_ "sea water temperature". So the "Sea Water Temperature" filter should be removed, and we should either stop harvesting that variable, or make sure we don't label it as sea water temperature.

**Comment by leonardolaiolo** [comment]
I deleted the 2 filters, here the [PR](https://github.com/aodn/geoserver-config/pull/717). 
I understand that to close this issue some modification in the harvester are required, with 2 possible options: 
option 1 - stop harvesting "TEMP" variable in files from ADCP instruments
option 2 - relabel "TEMP" variable
@lbesnard or @mhidas I could have a look into the harvester, any suggestion if go for option 1 or 2?

**Comment by lbesnard** [comment]
i don't think it is necessary to not harvest the TEMP variable. It will be just a bonus variable

**Comment by leonardolaiolo** [comment]
Cannot promote to prod until portal-rc get fixed (layers not available on portal-rc). Issue already open [here](https://github.com/aodn/issues/issues/1049).
